app/code/Magento/ImportExport/Test/Unit/Model/Import/Source/FileParser/_files/test_options.csv

```diff
@@ -0,0 +1 @@
+column1|;column2;|column3
```

Comment:
There are no component level integration tests in Magento 2 repository and the one you are referring to requires to work with real database and other components enabled. However, this test case is quite isolated and does not leave any side effects visible to other test cases. Unit scope for this one is parsing a ZIP file, but integration test is more about testing integration of components versus just I/O operation.